forest/jasmine/traj2stats.py

```diff
@@ -473,21 +474,45 @@ def gps_summaries(
             for pause in pause_array:
                 if places_of_interest is not None:
                     all_place_probs = [0] * len(places_of_interest)
-                    pause_circle = transform_point_to_circle(
-                        pause[0], pause[1], person_point_radius
-                    )
+                    if (
+                        f"{pause[0]}, {pause[1]} - person"
```

Comment:
Could you make this a variable? I think it would make the code less repetitive and more readable.